instructions updated (#507)
kubeshop/testkube: docs/installing.md

```sh
bash < <(curl -sSLf https://kubeshop.github.io/testkube/install.sh )
```
Alternative install:

1. Download binary of your choice
2. Upack it (tar -zxvf testkube_0.6.5_Linux_arm64.tar.gz)
3. Move it to location in the PATH for example `mv  testkube_0.6.5_Linux_arm64/kubectl-testkube /usr/local/bin/kubectl-testkube`

For Windows download desired binary from [here](https://github.com/kubeshop/testkube/releases), unpack the binary and add it to `%PATH%`. 

We have plans to build installers for most popular OS and system distros.

## Install `testkube` components in your cluster

The testkube kubectl plugin provides an install command to install testkube in your cluster. 

Run 
```shell
kubectl testkube install
```

> Above command will install following components in your Kubernetes cluster:

Reviewer: Maybe here you can provide an example how the components should look with `kubectl get all -n testkube`.

Author: take another look i extended it

1. Download binary with version of your choice (recent one is recommended)
2. Upack it (tar -zxvf testkube_0.6.5_Linux_arm64.tar.gz)

> 3. Move it to location in the PATH for example `mv  testkube_0.6.5_Linux_arm64/kubectl-testkube /usr/local/bin/kubectl-testkube`

Reviewer 2: ```suggestion
3. Move it to a location in the PATH. For example `mv  testkube_0.6.5_Linux_arm64/kubectl-testkube /usr/local/bin/kubectl-testkube`
```

> For Windows download desired binary from [here](https://github.com/kubeshop/testkube/releases), unpack the binary and add it to `%PATH%`. 

Reviewer 2: ```suggestion
For Windows, download the binary from [here](https://github.com/kubeshop/testkube/releases), unpack the binary and add it to `%PATH%`. 
```

Desired is implied

> We have plans to build installers for most popular OS and system distros [#161](https://github.com/kubeshop/testkube/issues/161).

Reviewer 2: ```suggestion
We have plans to build installers for the most popular Operating Systems and system distros [#161](https://github.com/kubeshop/testkube/issues/161).
```

## Install `testkube` components in your cluster

The testkube kubectl plugin provides an install command to install testkube in your cluster. 

Run 
```shell
kubectl testkube install
```

> Above command will install following components in your Kubernetes cluster:

Reviewer 2: ```suggestion
The above command will install the following components in your Kubernetes cluster:
```

1. Testkube API
2. `testkube` namespace
3. CRD for scripts 
4. MongoDB
5. Minio - optional

You can confirm it by running:
```
$ kubectl get all -n testkube
NAME                                       READY   STATUS    RESTARTS   AGE
pod/testkube-api-server-5478577b5b-jnnv6   1/1     Running   0          64s
pod/testkube-mongodb-5d95f44fdd-8wkwh      1/1     Running   0          64s

NAME                          TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)          AGE
service/testkube-mongodb      ClusterIP   10.43.192.11   <none>        27017/TCP        64s
service/testkube-api-server   NodePort    10.43.32.229   <none>        8088:31868/TCP   64s

NAME                                  READY   UP-TO-DATE   AVAILABLE   AGE
deployment.apps/testkube-api-server   1/1     1            1           64s
deployment.apps/testkube-mongodb      1/1     1            1           64s

NAME                                             DESIRED   CURRENT   READY   AGE
replicaset.apps/testkube-api-server-5478577b5b   1         1         1       64s
replicaset.apps/testkube-mongodb-5d95f44fdd      1         1         1       64s
```

> By default testkube is installed in `testkube` namespace.

Reviewer 2: ```suggestion
By default testkube is installed in the `testkube` namespace.
```

| api-server.image.repository        | yes         | "kubeshop/testkube-api-server"     |
| api-server.image.pullPolicy        | yes         | "Always"                           |
| api-server.image.tag               | yes         | "latest"                           |
| api-server.service.type            | yes         | "NodePort"                         |
| api-server.service.port            | yes         | 8088                               |
| api-server.mongoDSN                | yes         | "mongodb://testkube-mongodb:27017" |
| api-server.telemetryDisabled       | yes         | false                              |
| api-server.storage.endpoint        | yes         | localhost:9000                     |
| api-server.storage.accessKeyId     | yes         | minio                              |
| api-server.storage.accessKey       | yes         | minio123                           |
| api-server.storage.scrapperEnabled | yes         | false                              |

>For more configuration parameters of `MongoDB` chart please look here:
https://github.com/bitnami/charts/tree/master/bitnami/mongodb#parameters

## Uninstall `testkube`

> You can uninstall TestKube using uninstall command integrated into testkube plugin. 

Reviewer 2: ```suggestion
You can uninstall TestKube using the uninstall command integrated into the testkube plugin. 
```

```
kubectl testkube uninstall [--remove-crds]
```

> Optionally you can use `--remove-crds` flag which clean all installed Custom Resource Definitions installed by TestKube.

Reviewer 2: ```suggestion
Optionally you can use the `--remove-crds` flag which will clean all installed Custom Resource Definitions installed by TestKube.
```
